src/issue.js

```diff
@@ -116,10 +116,40 @@ async function closeIssue(token, issueNo) {
   logInfo(`Closed issue no. - ${issueNo}`)
 }
 
+async function isSnoozed(token, latestReleaseDate, staleDate) {
+  const octokit = github.getOctokit(token)
+  const { owner, repo } = github.context.repo
+  const { data: closedNotifyIssues } = await octokit.request(
+    `GET /repos/{owner}/{repo}/issues`,
+    {
+      owner,
+      repo,
+      creator: 'app/github-actions',
+      state: STATE_CLOSED,
+      sort: 'created',
+      state_reason: 'not_planned',
+      direction: 'desc',
+      labels: ISSUE_LABEL,
+      since: latestReleaseDate,
+    }
+  )
+
+  if (!closedNotifyIssues?.length) {
+    return false
+  }
+
+  if (isStale(closedNotifyIssues[0].closed_at, staleDate)) {
+    return false
+  }
+
+  return true
```

Comment:
return isStale(...)

Reply:
I've preferred to write the code extensively because this is the tricky part, so i wanted it to be as clear as possible, if necessary I can one line it